components/autosize_textarea.tsx

```diff
@@ -108,6 +108,7 @@ export class AutosizeTextarea extends React.PureComponent<Props> {
         Reflect.deleteProperty(props, 'providers');
         Reflect.deleteProperty(props, 'channelId');
         Reflect.deleteProperty(props, 'forwardedRef');
+        Reflect.deleteProperty(props, 'onWidthChange');
```

Comment:
`onWidthChange` was getting passed to textarea

<img width="906" alt="Screenshot 2023-02-16 at 19 11 41" src="https://user-images.githubusercontent.com/1501599/220289912-6293f733-6a6b-4969-853d-a0f969010453.png">